tmpl/fragment_callback.mustache

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portier &ndash; Forwarding...</title>
+    <script src="/static/fragment_callback.js"></script>
```

Comment:
If adding the event check like mentioned above this script tag could get the async attribute.